yocto-base/00_flash_and_verify.robot

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Library             Collections
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../../sonoff-rest-api/sonoff-api.robot
+Resource            ../../rtectrl-rest-api/rtectrl.robot
+Resource            ../../variables.robot
+Resource            ../../keywords.robot
+Resource            ../lib/sd-wire.robot
+
+Suite Setup         Prepare Test Suite
+Suite Teardown      Log Out And Close Connection
+
+
+*** Test Cases ***
+Verify number of connected SD Wire devices
+    [Documentation]    Test confirms the number of connected SD Wire devices to
+    ...    the DUT platform.
+    ${sd_wire_id_list}=    Get List Of SD Wire Ids
+    ${length_of_sd_wire_id_list}=    Get Length    ${sd_wire_id_list}
+    # we are currently doing this because we only expect one SD Wire
```

Comment:
@WiktorG351 let's rephrase this comment, to include recent changes supporting multiple connections and value specified in the device config

Reply:
@artur-rs fixed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/22/commits/0639db54fa4d298b32ad5d3106e4e34db1493cc1)